content/tutorials/grafana-fundamentals.md

```diff
@@ -106,216 +119,234 @@ To the far left you can see the _sidebar_, a set of quick access icons for navig
+((Just above, we list the parts of an alert as alert rules and notification channels. If possible, we should perform the tasks in the same order. If not possible, then we should rewrite the points above so they make sense in the reverse order.))
+In this step, you'll send alerts using _web hooks_. To test your alerts, you first need to have a place to send them.
 
 - Browse to [requestbin.com](https://requestbin.com)
-- Unselect **Private (requires log in)**, and click **Create Request Bin**.
-- You request bin is now waiting for the first request.
+- Clear the **Private (requires log in)** check box, then click **Create Request Bin**.
+  You request bin is now waiting for the first request.
 - Copy the endpoint URL.
 
-Next, you'll configure a notification channel for web hooks, to send notifications to your Request Bin:
+Next, configure a notification channel for web hooks to send notifications to your Request Bin.
 
-- In the side bar, click **Alerting** -> **Notification channels**.
+- In the side bar, hover your cursor over the **Alerting** (bell) icon and then click **Notification channels**. ((When I did this, Grafana asked if I wanted to Save my time range. I clicked Discard, hope that was the right choice. ))
 - Click **Add channel**.
-- In the **Name** box, type RequestBin.
-- In the **Type** box, select "webhook".
-- In the **Url** box, paste the endpoint to your request bin.
+- In **Name**, type **RequestBin**.
+- In **Type**, select **webhook**.
+- In **Url**, paste the endpoint to your request bin.
 - Click **Send Test** to send a test alert to your request bin.
+- Click **Save**. Grafana displays a list of your existing notification channels.
+  ((Okay...now what? What should I see?))
 
 ### Configure an alert rule
 
 Now that Grafana knows how to notify you, it's time to set up an alert rule:
 
-- Go to the dashboard you just created, and **Edit** the Traffic panel.
+- Go to the dashboard you just created. ((Need instructions for how to navigate there from the Notification channels list.))
+- Click the Traffic panel title and then type **e**. This keyboard shortcut opens the panel in edit mode.
 - Click the bell icon to the left of the panel editor to access the settings for alerting.
-- In the **Name** box, type "My alert".
-- In the **Evaluate every** box, type "5s". For the purpose of this tutorial, the evaluation interval is intentionally shorter than what's recommended, to make it easier to test.
-- In the **For** box, type "0m".
-- Change the alert condition to:
+- Click **Create Alert**.
+- In **Name**, enter **My alert**.
+- In **Evaluate every**, type **5s**. For the purpose of this tutorial, the evaluation interval is intentionally shorter than what's recommended to make it easier to test. ((This begs the question, what is recommended? Where would I find that?))
+- In **For**, enter **0m**. ((What does 0m do? Make it last forever?))
```

Comment:
With a For value of, say 5m, the alert is first set to a pending state for 5 minutes. If the alert rule is still triggering by the end of the interval, we trigger the alert.

Reply:
We should probably explain that, because "0m" is super confusing.